third_party/3/jwt/algorithms.pyi

```diff
@@ -0,0 +1,2 @@
+class Algorithm:
```

Comment:
Shouldn't this be `Algorithm: Any`? That way mypy won't produce spurious errors if you use one of the methods on an Algorithm class.